src/functionalTest/java/uk/gov/hmcts/sptribs/caseworker/CaseworkerBundleStitchCompleteFT.java

```diff
@@ -0,0 +1,29 @@
+package uk.gov.hmcts.sptribs.caseworker;
+
+import io.restassured.response.Response;
+import org.junit.jupiter.api.Test;
+import org.springframework.boot.test.context.SpringBootTest;
+import uk.gov.hmcts.sptribs.caseworker.util.EventConstants;
+import uk.gov.hmcts.sptribs.testutil.FunctionalTestSuite;
+
+import java.util.Map;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.springframework.http.HttpStatus.OK;
+import static uk.gov.hmcts.sptribs.testutil.CaseDataUtil.caseData;
+import static uk.gov.hmcts.sptribs.testutil.TestConstants.ABOUT_TO_SUBMIT_URL;
+
+@SpringBootTest
+public class CaseworkerBundleStitchCompleteFT extends FunctionalTestSuite {
+
+    private static final String REQUEST = "classpath:request/casedata/ccd-callback-casedata-caseworker-stich-bundle.json";
+
+    @Test
+    public void shouldRecieveOkResponseWhenAboutToSubmitCallbackIsInvoked() throws Exception {
+        final Map<String, Object> caseData = caseData(REQUEST);
+
+        final Response response = triggerCallback(caseData, EventConstants.ASYNC_STITCH_COMPLETE, ABOUT_TO_SUBMIT_URL);
```

Comment:
You can import the event name as static var removing the need for EventConstants.